tests/admin_widgets/tests.py

```diff
@@ -437,9 +441,14 @@ def test_render(self):
         self.assertHTMLEqual(
             w.render("test", datetime(2007, 12, 1, 9, 30)),
             '<p class="datetime">'
-            'Date: <input value="2007-12-01" type="text" class="vDateField" '
+            '<label for="">Date:</label> '
```

Comment:
That the for is blank doesn't feel like a good test, could this widget be rendered for a field to have a value in the for attribute?